components/net/image_cache.rs

```diff
@@ -415,6 +417,9 @@ impl ImageCacheStore {
 
 pub struct ImageCacheImpl {
     store: Arc<Mutex<ImageCacheStore>>,
+
+    // Thread pool for image decoding
```

Comment:
```suggestion
    /// Thread pool for image decoding
```

with `///` hovering overt `thread_pool ` in IDE show this comment

Reply:
Thanks for catching that!